### DISCUSSION TITLE
Elecrow ESP32 5" display

### DISCUSSION THREAD

**Discussion by ColinHMurdoch** [Bodmer/TFT_eSPI#3012] [New processor/display support or capability request]
I was recently tempted to purchase an Elecrow ESP32 display with ESP32-S3 built in.  I had viewed one or two videos that used this range of displays using the TFT_eSPI library as they as ILI9488 chip.  However I have now discovered that the larger displays use the ILI6122 & ILI5960 chips which are not supported.  Setting up the display using the LVGL graphics library seems overly complicated and to date I have not managed to upload anything functional to the screen.
Is it likely that support for these screens/chips will be added to the library in the future?

**Comment by Bodmer** [comment]
It is unlikely support will be added in the near future.

**Reply by GilchristT** [reply]
@Bodmer Just out of interest, why is it unlikely?

No criticism nor passive aggressiveness intended :-) I'm just interested in what the blocker is e.g. if it's because they're a completely different architecture, lack of drivers, lack of time (I'm money on that one), etc?

I'm always keen to learn (I don't even know what it means that they use two different display drivers)

**Comment by jalmince** [comment]
Hi Colin! same issue here... Have you found a solution ?
cheers,
Jpaul

**Comment by yubastard** [comment]
I'm in this exact same boat. It's proven soooo hard to get Squareline UI working! There has to be a way, else it's kinda useless to have this big beautiful display.

**Reply by techbear1980** [reply]
Same here.  These CrowPanel devices are supercheap now, and the screen is so big and cool.  But I can't get anything to work. :(

**Reply by yubastard** [reply]
After a long time I got it to work using LVGL 9.1 and LovyannGFX. I think it was the bar gradients that were taking up too much resources because I took them off the Squareline Vision UI and it worked! Though also ChatGPT cooked up some CPU thread code to keep the flush_cb and lv_timer_handler on the same thread, but I'm suspecting the color gradients were just too much for the resources at hand.
